docs/guides/using-with-flakes-devshell.md

```diff
@@ -0,0 +1,162 @@
+`devenv` is partly a CLI, but also a way to configure a development shell declaratively. This configuration can also be used without using the `devenv` CLI inside other Nix flakes.
```

Comment:
```suggestion
`devenv` can be used without the `devenv` CLI by integrating into [Nix Flakes](https://www.tweag.io/blog/2020-05-25-flakes/) , if you're more familiar with the Nix language/ecosystem.
```

The target audience for this guide are experienced Nix developers, so I want to make sure the first sentence catches that.